.github/workflows/docker-build.yml

```diff
@@ -12,11 +12,15 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        # The jekyll/builder is used by GitHub pages
-        docker-image: [ 'jekyll/builder', 'sylhare/jekyll' ]
+        # The jekyll/builder was replaced by the action jekyll-build-pages
+        docker-image: [ 'sylhare/jekyll' ]
 
     steps:
       - uses: actions/checkout@v4
+      - name: Build
+        uses: actions/jekyll-build-pages@v1
```

Comment:
Seems like this is more resilient than the jekyll/builder image that was used before.
Let's try it out instead of the docker image to make sure the build pass on gem updates.